pint/pint_convert.py

```diff
@@ -171,7 +172,8 @@ def convert(u_from, u_to=None, unc=None, factor=None):
     if factor:
         q *= ureg.Quantity(factor)
         nq *= ureg.Quantity(factor).to_base_units()
-    prec_unc = use_unc(nq.magnitude, fmt, args.prec_unc)
+    if args.unc:
```

Comment:
There is a place in this file where this logic happens:

```
if args.unc:
    import uncertainties
```

Perhaps you could put the `import` here, with a `try except` that will print a nice user friendly error asking the user to install `uncertainties` and `numpy`?

Reply:
Thanks @doronbehar ! 

I'll add it. Currently it will raise `Import` error, which I assumed was enough for people to understand and install `uncertainties`.

I think I understand the problem that you encountered is an issue with `uncertainties`  package not with `pint`. In my understanding `uncertainties` has `numpy` as optional dependency so even if you import `uncertainties` here in pint, the calculations seems to doesn't work.  The problem is basically [here](https://github.com/lmfit/uncertainties/blob/1238d37c6ed87d018e696589716de488e7d9eca7/uncertainties/core.py#L108), at this place the `uncertainties` should raise an warning not pass it.

Reply:
I have added an expection to be raised if `uncertainties` cannot be imported.